platform/aion/api.go

```diff
@@ -0,0 +1,58 @@
+package aion
+
+import (
+	"github.com/trustwallet/blockatlas/platform/aion/source"
+	"github.com/trustwallet/blockatlas/models"
+	"github.com/trustwallet/blockatlas/coin"
+	"github.com/trustwallet/blockatlas/util"
+	"github.com/sirupsen/logrus"
+	"github.com/gin-gonic/gin"
+	"github.com/spf13/viper"
+	"net/http"
+	"strconv"
+	"fmt"
+)
+
+var client = source.Client{
+	HttpClient: http.DefaultClient,
+}
+
+func Setup(router gin.IRouter) {
+	router.Use(util.RequireConfig("aion.api"))
+	router.Use(func(c *gin.Context) {
+		client.RpcUrl = viper.GetString("aion.api")
+		c.Next()
+	})
+	router.GET("/:address", getTransactions)
+}
+
+func getTransactions(c *gin.Context) {
+	trxs, err := client.GetTxsOfAddress(c.Param("address"))
+
+	if err != nil {
+		logrus.WithError(err).Errorf("Aion: Failed to get transactions for address %v", c.Param("address"))
+	}
```

Comment:
add integration tests

Reply:
Yes, please add an Aion address to the map in the integration test file https://github.com/TrustWallet/blockatlas/blob/master/test/main.go